Pelias/Pelias.h

```diff
@@ -0,0 +1,19 @@
+//
+//  Pelias.h
```

Comment:
Is this file actually needed? I know it's added by default when adding a framework target but seeing as it's a Swift-only framework, we shouldn't need an umbrella header. I can't find anything indicating either way on the 'Net.